### ISSUE TITLE
quantified self devices

### ISSUE THREAD

**Issue by yosemitebandit** [cyrusstoller/StartX-MedIC#8]
more hardware:

a wearable, quantified-self device that does one or both of the following:
- takes a picture of what you're looking at every few moments.  the photos are processed to determine your sociability via face-counting
- records your conversations, transcribes the text and performs some sentiment analysis to determine how your day is going..

this project be really tongue-in-cheek..  the idea of an _employer_ providing this is flat Orwellian.  but once again, it'd be a cool thing to builddddd..

**Comment by cyrusstoller** [comment]
Kinda like signing everyone up for the original justin.tv?

**Comment by wgpatrick** [comment]
Sorry for being slow... but do we have to build something an employer would provide? That seems sort of limiting.

**Comment by yosemitebandit** [comment]
yeah, the [about section](http://www.startxmed.com/about.html) makes it sound like we're pitching stuff to companies that will help their employees be healthier.  it's not really a straight health hackathon, per se..

but I've got no problem with ignoring that sentiment and just spending a weekend building something cool..

**Comment by jasdeepg** [comment]
my opinion is that i'd rather stay on script. we did apply and they did
accept us with the premise that we'd be competing.

just my 2 cents.

**Comment by wgpatrick** [comment]
i don't really care that much about staying on script personally. main goal
for me is to make something interesting.

**Comment by cyrusstoller** [comment]
i agree with @jasdeepg. i'd prefer to stay on script as well.
